flask_app/api/map_paths.py

```diff
@@ -11,7 +11,7 @@ def correct_action(current_state):
    elif current_state == 'prepare':
       actions = ['actions/split_data.html']
    elif current_state == 'split':
-      actions = ['actions/train_model.html']
+      actions = ['actions/train_lin_reg.html']
```

Comment:
Be careful renaming variables without changing the name of the file that it is accessing. This is what caused the `error TemplateNotFound: actions/train_lin_reg.html` when going from split to train.

Reply:
I didn't add anything to the flask app that would create a page for it I simply renamed train_model to train_lin_reg everywhere it appeared in the code itself so I'm guessing the class name is still train_model.html, but I can change that real quick. As it stands though, I don't believe there is anything connecting flask to the model evaluation so if the code is generated through the flask app, then the evaluation won't be added.